docs/configuration.md

```diff
@@ -163,5 +164,13 @@ Key | Description | Type | Default | Required
 `carriers` | List of losing carriers to select from on port request submission. | `Array` | | `false`
 `carriers.[]` | Name of losing carrier (should be unique). | `String` | | `false`
 
+#### `provisioner`
+
+Key | Description | Type | Default | Required
+--- | --- | :---: | :---: | :---:
+`brands` | Map custom settings individually by device brand | `Object` | | `false`
+`brands.[]` | Each object key correspont to the device brand name (`yealink`, `avaya`...) | `Object` | | `false`
+`brands.[].keyFunctions` | List key functions available per brand | `Array` | | `false`
```

Comment:
Since there is no documentation entry about available functions, let's add a link to the related schema so ops get an idea of possibilities:
```suggestion
`brands.[].keyFunctions` | List key functions available per brand ([schema][combo-keys-schema]). | `Array` | | `false`
```
and add the link reference at the bottom of the file:
```md
[combo-keys-schema]: https://github.com/2600hz/kazoo/blob/master/applications/crossbar/priv/couchdb/schemas/devices.combo_key.json
```